### PR TITLE
Remove all ^L

### PR DESCRIPTION
**Pull request by guicho271828**

### CODE REVIEW

**Comment by joaotavora** [comment]
Hmm, no.  ^L are page markers useful by Emacs.

You're opening so many issues.  Sure you don't want to write a mail and discuss stuff more freely?  Appreciate you're trying out SLY.

**Comment by guicho271828** [comment]
I just make a separate issue/pr for a separate topic.

**Comment by joaotavora** [comment]
> I just make a separate issue/pr for a separate topic.

That's good, indeed, so thanks! Many people will mix everything in the same issue, which is bad. But, on the other hand,  such a large number is hard to follow. it's easier to discuss many issues with the same person via email.
